PublishedModules/glueckself/mpdclient/mpdclient.install

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "mpdclient",
+	"version": 0.1,
+	"author": "glueckself",
+	"maintainers": ["glueckself"],
+	"desc": "Control an MPD server",
```

Comment:
@Psychokiller1888 maintainers are only additional people maintaining the module so the author does not need to appear in that least right?